test/integration/bridging-non-rebasable-new_L1-old_L2.integration.test.ts

```diff
@@ -0,0 +1,40 @@
+import { BigNumber } from 'ethers'
+import { wei } from "../../utils/wei";
+import { scenario } from "../../utils/testing";
+import { bridgingTestsSuit, ctxFactory } from "./_bridging-non-rebasable";
+
+bridgingTestsSuit(
+  scenario(
+    "Optimism :: Bridging X non-rebasable token integration test",
+    ctxFactory(
+      true,
```

Comment:
Can we make these constants more readable either introducing aliases or least with some comments? 💪